napari_animation/_tests/test_animation.py

```diff
@@ -110,6 +112,22 @@ def test_animate_filenames(
         assert saved_files == expected
 
 
+@pytest.mark.parametrize("ext", [".mp4", ".mov", ".avi", ""])
+def test_animation_file_metadata(animation_with_key_frames, tmp_path, ext):
+    """Test output video file contians napari version metadata()"""
+    animation = animation_with_key_frames
+    output_filename = tmp_path / f"test{ext}"
+    animation.animate(output_filename)
```

Comment:
I notice that the other tests (like `test_animate_filenames` above) are very careful to mock/patch the video writer, so I'm not sure if you won't like that it actually runs here.
I'm not really sure how best to test this without generating real output video files though.

Reply:
I think this is okay, that was done because the output wasn't being checked for anything afaik